src/app/services/node-actions.service.ts

```diff
@@ -32,16 +32,14 @@ import {
   ContentNodeSelectorComponent,
   ContentNodeSelectorComponentData,
   ContentNodeDialogService,
-  ShareDataRow
+  ShareDataRow,
+  NodeActionsEnum
 } from '@alfresco/adf-content-services';
 import { MinimalNodeEntity, MinimalNodeEntryEntity, SitePaging, NodeChildAssociationPaging, NodeChildAssociationEntry } from '@alfresco/js-api';
 import { ContentApiService } from '@alfresco/aca-shared';
 import { catchError, map, mergeMap } from 'rxjs/operators';
 
-export enum BatchOperationType {
-  copy = 'copy',
-  move = 'move'
-}
+type BatchOperationType = Extract<NodeActionsEnum, NodeActionsEnum.COPY | NodeActionsEnum.MOVE>;
```

Comment:
Are you still using this somewhere or can we remove it? In the code of this class you are using directly the NodeActionsEnum and the BatchOperationType is used only in line 536 `action: BatchOperationType` if I am not mistaken

Reply:
Indeed. I wrote it this way to restrict the values of the functions that use it to just COPY and MOVE, so that other values of the enumeration cannot be used in those functions. If this is deemed unnecessary, I can remove it and just use the NodeAction enumeration

Reply:
Hmm I think it's ok you have a point 🙂 Thnx for the clarification